src/Device/Driver/LX_EOS/LXEosDevice.hpp

```diff
@@ -259,11 +335,47 @@ class LXEosDevice : public AbstractDevice
    * @param env OperationEnvironment
    * @return true if ACK was received,
    * @return false if time out or other exception occurred
-   * @note timeout is 1 second
+   * @note timeout is 3 seconds
```

Comment:
?

Reply:
It returns false if ACK is not received within 3 seconds

Reply:
But why is the documentation changed in this commit? Did you change the timeout? Why?